pkg/chunked/tar_split_linux.go

```diff
@@ -0,0 +1,68 @@
+package chunked
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/vbatts/tar-split/archive/tar"
+	"github.com/vbatts/tar-split/tar/storage"
+)
+
+// iterateTarSplit calls handler for each tar header in tarSplit
+func iterateTarSplit(tarSplit []byte, handler func(hdr *tar.Header) error) error {
+	// This, strictly speaking, hard-codes undocumented assumptions about how github.com/vbatts/tar-split/tar/asm.NewInputTarStream
```

Comment:
In theory this API could land as a PR to that repo?

Reply:
That would certainly be better. The timing will probably force us to carry this in c/storage at for a few days/weeks, but I should definitely prepare a tar-split PR.

Reply:
I have recorded the task to file a tar-split PR as an item on https://github.com/containers/image/issues/2189 .